contact.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+
+<html lang="en-GB">
+
+	<head>
+		<title>Contact Page</title>
+		<link rel="stylesheet" type="text/css" href="portfolio_style.css">
+	</head>
+		
+	<body>
+		
+		<nav class="nav_bar">
+			<p>Karen Schulz</p>
+			
+			<div>
+				<a href="portfolio_home.html">Home</a>
```

Comment:
I assume that this "portfolio_home.html" is your home page?
If it is, then it should be named index.html as this is standard practice =]

Reply:
Yes, thanks Mike. I changed it to index.html as you suggested. You are right.